src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -202,18 +244,10 @@ ss::future<> service::configure() {
       _ctx.smp_sg, [has_ephemeral_credentials](service& s) {
           s._has_ephemeral_credentials = has_ephemeral_credentials;
       });
-    co_await _controller->get_security_frontend().local().create_acls(
-      {security::acl_binding{
-        security::resource_pattern{
-          security::resource_type::topic,
-          model::schema_registry_internal_tp.topic,
-          security::pattern_type::literal},
-        security::acl_entry{
-          principal,
-          security::acl_host::wildcard_host(),
-          security::acl_operation::all,
-          security::acl_permission::allow}}},
-      5s);
+
+    ssx::background = ssx::spawn_with_gate_then(_gate, [this] {
```

Comment:
issue: I think it would be preferable to not start the SR listeners until it's ready.

Would it be preferable to keep all of SR startup blocked until this succeeds, and then do backgrounding in application.cc?

Reply:
> Would it be preferable to keep all of SR startup blocked until this succeeds

Blocking all of SR startup with a potential infinite loop risks blocking the entire broker. https://github.com/redpanda-data/redpanda/pull/11563#discussion_r1248002435

> then do backgrounding in application.cc?

I am confused by this statement. If we brought the retry loop into the foreground, then there is nothing to put into the background.

Reply:
>  I think it would be preferable to not start the SR listeners until it's ready.

Why is this preferable? If we kept the retry loop in the background, what does it break?

Reply:
Spoke in standup about this. We decided to keep the retry loop in the background for now and address SR and PP startup at another time. I created an epic for refactoring SR and PP startup here https://github.com/redpanda-data/core-internal/issues/684